torch/_refs/__init__.py

```diff
@@ -1972,6 +1973,26 @@ def _reduction(
     return result
 
 
+def _make_copy_from_view(fn):
+    # Does not handle multiple outputs, but I think we don't have any such an op
+    name = fn.__name__
+    fn = out_wrapper()(fn)
+
+    def _fn(*args, out=None, **kwargs):
+        result = fn(*args, **kwargs)
+        if out is None:
+            return result.clone()
+        else:
+            _maybe_resize_out(out, result.shape)
+            _safe_copy_out(copy_from=result, copy_to=out, exact_dtype=False)  # type: ignore[arg-type]
+            return out
+
+    copy_name = f"{name}_copy"
+    _fn.__name__ = copy_name
```

Comment:
Note for our future selves: we should probably create a standard for wrappers like this. Here we're updating the name but `out_wrapper` is more aggressive:

https://github.com/pytorch/pytorch/blob/75a5a46aa005e1c56f5a7935003cb480f33f9257/torch/_prims_common/wrappers.py#L264